consensus/Cargo.toml

```diff
@@ -14,6 +14,7 @@ serialization = { path = "../serialization" }
 utils = { path = "../utils" }
 utxo = { path = '../utxo' }
 
+parity-scale-codec.workspace = true
```

Comment:
Normally you shouldn't have to add this. Just adding the serialization crate should be enough.

Reply:
I thought so too, but if removed then it throws errors during compilation.